mon-entreprise/source/rules/conventions-collectives/spectacle-vivant.yaml

```diff
@@ -92,9 +92,9 @@ contrat salarié . intermittents du spectacle . retraite complémentaire technic
       - statut cadre
       - technicien
   remplace:
-    - règle: retraite complémentaire . taux employeur tranche 1
+    - règle: retraite complémentaire . employeur . taux tranche 1
```

Comment:
Top ça ! Le seul endroit où on utilise encore les `[ref monnom]` c'est pour les tranches de barèmes, car on ne peut donner un nom à une tranche complète, mais c'est une lacune qu'il faudra lever aussi, cf. par exemple le publicodes du calcul de l'IS qui est trop répétitif https://github.com/betagouv/mon-entreprise/pull/1230/files#diff-77ed7449d170c4b42708801c419bad20a8228c12aa96ec98945447fb7c2e0d25